conftest.py

```diff
@@ -2,6 +2,14 @@
 from enum import Enum
 from supplier import models
 from client.models import Client
+from supplier_product.models import SupplierProduct
+from product.tests import test_product
+
+
+prod = test_product.product0
+ID = 123685
+QUANTITY = 9
+PRICE = 14
```

Comment:
The names used here are too broad to be used inside this file which is used in the context of all modules in the repo. The names should be more specific, e.g. `SUPPLIER_PRODUCT_ID`.

Regardless, it will be better to have test constants as fixtures rather than variables. Having them as fixtures makes sure that they are regenerated with the assigned value for every test case. When they are global variables, one test could modify them, consequentially making following tests to fail.

example:
```
@pytest.fixture
def supplier_product_id():
    return 123685

...

@pytest.fixture
def supplier_product0(prod, saved_supplier0, supplier_product_id):
    prod.save()
    return SupplierProduct(
        supplier_product_id=supplier_product_id,
        qr_code=prod,
        user_name=saved_supplier0,
        price=PRICE,
        quantity=QUANTITY
    )
```